src/main/java/com/google/sps/servlets/ScheduledInterviewServlet.java

```diff
@@ -169,6 +171,68 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     }
   }
 
+  // Send the request's contents to Datastore in the form of an updated ScheduledInterview object.
+  // Adds a shadow.
```

Comment:
Nit: specify it adds current user as the shadow

Reply:
Done